services/ui-src/src/utils/reports/entities.ts

```diff
@@ -141,6 +141,195 @@ export const getFormattedEntityData = (
+            geomappingInfo.push(
+              `Q1 (optional): ${findNestedData(
+                geomappingKeys,
+                "_q1ActualMaxDist"
+              )}`
+            );
+          findNestedData(geomappingKeys, "_q2ActualMaxDist") &&
+            geomappingInfo.push(
+              `Q2 (optional): ${findNestedData(
+                geomappingKeys,
+                "_q2ActualMaxDist"
+              )}`
+            );
+          findNestedData(geomappingKeys, "_q3ActualMaxDist") &&
+            geomappingInfo.push(
+              `Q3 (optional): ${findNestedData(
+                geomappingKeys,
+                "_q3ActualMaxDist"
+              )}`
+            );
+          findNestedData(geomappingKeys, "_q4ActualMaxDist") &&
+            geomappingInfo.push(
+              `Q4 (optional): ${findNestedData(
+                geomappingKeys,
+                "_q4ActualMaxDist"
+              )}`
+            );
+
+          // percent of enrollees annually
+          findNestedData(geomappingKeys, "_annualPercentMetStandard") &&
+            geomappingInfo.push(
+              `Annual (optional): ${findNestedData(
+                geomappingKeys,
+                "_annualPercentMetStandard"
+              )}`
+            );
+          findNestedData(geomappingKeys, "_annualPercentMetStandardDate") &&
+            geomappingInfo.push(
+              `Date of analysis of annual snapshot (optional): ${findNestedData(
+                geomappingKeys,
+                "_annualPercentMetStandardDate"
+              )}`
+            );
+
+          // actual max time annually
+          findNestedData(geomappingKeys, "_annualMaxTime") &&
+            geomappingInfo.push(
+              `Annual (optional): ${findNestedData(
+                geomappingKeys,
+                "_annualMaxTime"
+              )}`
+            );
+          findNestedData(geomappingKeys, "_annualMaxTimeDate") &&
+            geomappingInfo.push(
+              `Date of analysis of annual snapshot (optional): ${findNestedData(
+                geomappingKeys,
+                "_annualMaxTimeDate"
+              )}`
+            );
+
+          // actual max distance annually
+          findNestedData(geomappingKeys, "_actualMaxDistance") &&
+            geomappingInfo.push(
+              `Annual (optional): ${findNestedData(
+                geomappingKeys,
+                "_actualMaxDistance"
+              )}`
+            );
+          findNestedData(geomappingKeys, "_actualMaxDistanceDate") &&
+            geomappingInfo.push(
+              `Date of analysis of annual snapshot (optional): ${findNestedData(
+                geomappingKeys,
+                "_actualMaxDistanceDate"
+              )}`
+            );
+
+          // add in to analysis methods array
+          analysisMethodsUsed.splice(
+            indexOfGeomapping + 1,
+            0,
+            ...geomappingInfo
+          );
+        }
```

Comment:
insert all the new `label: value` texts we made into the `analysisMethodsUsed` array, which is ultimately paired into a Q:A object below for display